<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDialog_setting</class>
 <widget class="QDialog" name="QDialog_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QDialog_setting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,10">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>设置区域</string>
     </property>
     <layout class="QVBoxLayout" name="_2" stretch="1,10">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="pB_grab">
        <property name="text">
         <string>采集</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vLayout_param"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,100">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,100">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pB_sendClick">
           <property name="text">
            <string>点击</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>x：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_x"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>y：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lE_y"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>图片区域</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="ckB_adjReg">
          <property name="text">
           <string>适应区域</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="sA_pic">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>698</width>
             <height>562</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
